modules/report/includes/commerce_pos_report.pages.inc

```diff
@@ -286,7 +286,7 @@ function commerce_pos_report_end_of_day($form, &$form_state) {
   if (!empty($totals)) {
     $js_settings['currencies'] = commerce_pos_report_currency_js(array_keys($totals));
 
-    $form['results']['#attached']['js'][] = drupal_get_path('module', 'commerce_pos_report') . '/js/commerce_pos_report.js';
```

Comment:
There is something wrong with my local Git setup seems like there are files which are not getting tracked, I already updated this file to use sorter.js , I removed old one as it was using #(css Id selectors) limiting reusability. I will be double checking this and pushing the file back .